### DISCUSSION TITLE
how to move to left file explorer

### DISCUSSION THREAD

**Discussion by helloYeon** [nvim-tree/nvim-tree.lua#1563] [Q&A] [answered]
### Q1 how to move to left file explorer?
　↑、[Ctrl + H] is Not working in my pc(iterm2)

### Q2 how to move to right file editor?　
　↑、[Ctrl + I] is Not working in my pc(iterm2)

 ![image](https://user-images.githubusercontent.com/50859345/187334107-fcd68b10-9618-42a6-9ac4-e8fd361da284.png)

plz,answer

**Comment by alex-courtis** [comment, marked as answer]
`:help index` shows

```
|CTRL-H|	CTRL-H		1  same as "h"
|CTRL-I|	CTRL-I		1  same as <Tab>
```

If you wish to navigate windows you can use the `<C-W>` family, see `:help ctrl-w`.

In your case you can move to the nvim-tree window with `<C-W>h` and to the buffer with `<C-W>l`.

**Reply by helloYeon** [reply]
@alex-courtis 

i resoloved, thank ! have a nice day
